modules/bootkube/resources/bootkube.sh

```diff
@@ -90,15 +105,14 @@ set -e
 
 echo "etcd cluster up. Killing etcd certificate signer..."
 
-podman kill "$SIGNER"
-rm /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
-
-cp --recursive "$PWD/bootstrap-configs" /etc/kubernetes/bootstrap-configs
+podman rm --force etcd-signer
+rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
```

Comment:
[Only `-f` is in POSIX][1].

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/rm.html

Reply:
For RHCOS-only scripts, should we care? For developer scripts, I agree that we should stick with POSIX.